distribution/changelog.txt

```diff
@@ -4,6 +4,7 @@
 - Fix: [#12694] Crash when switching ride types with construction window open.
 - Fix: [#12701] Silent NSIS setup flag /S isn't silent, upgrade pop-up appears anyway.
 - Fix: Incomplete loop collision box allowed overlapping track (original bug).
+- Fix: [#12737] Space Rings draw the same vehicle 4 times.
```

Comment:
Sorry to be the bearer of bad news, but we keep the numbered fixes above the unnumbered ones.

Reply:
Oh, alrighty. Not a problem

Reply:
Oh shoot, didn't see that you also fixed it 😬